### PR TITLE
Mesa plateau F

### PR DESCRIPTION
**Pull request by NiLSPACE**
Implemented correct Mesa Plateau F(M) generation

### CODE REVIEW

File: src/Generating/CompoGenBiomal.cpp [hunk 1]
```diff
@@ -542,6 +542,20 @@ class cCompoGenBiomal :
...
+		EMCSBiome MesaVersion = a_ChunkDesc.GetBiome(a_RelX, a_RelZ);
+		if ((MesaVersion == biMesaPlateauF) || (MesaVersion == biMesaPlateauFM))
+		{
+			if (Top < 95 + static_cast<int>(m_MesaFloor.CubicNoise2D(NoiseY * 2, NoiseX * 2) * 6))
+			{
+				return;
+			}
+
+			BLOCKTYPE Block = m_MesaFloor.CubicNoise2D(NoiseX * 4, NoiseY * 4) < 0 ? E_BLOCK_DIRT : E_BLOCK_GRASS;
+			NIBBLETYPE Meta = Block == E_BLOCK_GRASS ? 0 : 1;
```
Comment (madmaxoft) [inline src/Generating/CompoGenBiomal.cpp]: Add a parenthesis around the comparison

Comment (NiLSPACE) [inline src/Generating/CompoGenBiomal.cpp]: For both the block and the meta?

Comment (madmaxoft) [inline src/Generating/CompoGenBiomal.cpp]: Of course. Both comparisons deserve it.

Comment (NiLSPACE) [inline src/Generating/CompoGenBiomal.cpp]: Done.

**Pull request merged**